tests/test_diffraction_objects.py

```diff
@@ -232,13 +238,40 @@ def test_diffraction_objects_equality(inputs1, inputs2, expected):
     assert (diffraction_object1 == diffraction_object2) == expected
 
 
-def _test_valid_diffraction_objects(actual_diffraction_object, function, expected_array):
```

Comment:
is there a reason this is getting deleted?

Reply:
It's not used anymore. It was originally designed for testing q_to_tth and tth_to_q.